packages/ssz/test/perf/iterate.test.ts

```diff
@@ -55,16 +54,14 @@ describe("readonly values - iterator vs array", () => {
   });
 });
 
-// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
 function createBalanceList(count: number) {
-  const VALIDATOR_REGISTRY_LIMIT = 1099511627776;
```

Comment:
why is the casing changed here? this is supposed to be `VALIDATOR_REGISTRY_LIMIT` constant from CL spec

Reply:
then we need to move it to global scope, using camelCase here is wrong

Reply:
there were a few things that changed to camel from constant case.  I missed that they are spec variables.  I agree those all should be moved back to const case to match the spec

Reply:
Done